src/main/java/gregtech/api/gui/widgets/CycleButtonWidget.java

```diff
@@ -132,7 +132,14 @@ public void readUpdateInfo(int id, PacketBuffer buffer) {
     public boolean mouseClicked(int mouseX, int mouseY, int button) {
         super.mouseClicked(mouseX, mouseY, button);
         if (isMouseOverElement(mouseX, mouseY)) {
-            this.currentOption = (currentOption + 1) % optionNames.length;
+            //Allow only the RMB to reverse cycle
+            if(button == 1) {
```

Comment:
Please make this one constant for readability sake

Reply:
I have moved this to a constant